hbase_sql/src/main/java/com/splicemachine/serializer/SpliceKryoSerializerInstance.java

```diff
@@ -26,7 +26,7 @@
 public class SpliceKryoSerializerInstance extends KryoSerializerInstance {
 
     public SpliceKryoSerializerInstance(KryoSerializer ks) {
-        super(ks);
+        super(ks,false);
```

Comment:
Sorry for the delay, it fails with an underflow.  I think it is caused by some optimization in not encoding but I did not have time to chase it down.